Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1168,20 +1108,29 @@ namespace AtomSampleViewer
             return;
         }
 
+        auto io = AZ::IO::LocalFileIO::GetInstance();
         screenshotTestInfo.m_toleranceLevel = *toleranceLevel;
+        static constexpr float ImperceptibleDiffFilter = 0.01;
 
-        if (screenshotTestInfo.m_officialBaselineScreenshotFilePath.empty())
+        if (screenshotTestInfo.m_officialBaselineScreenshotFilePath.empty()
+            || !io->Exists(screenshotTestInfo.m_officialBaselineScreenshotFilePath.c_str()))
         {
             ReportScriptError(AZStd::string::format("Screenshot check failed. Could not determine expected screenshot path for '%s'", screenshotTestInfo.m_screenshotFilePath.c_str()));
             screenshotTestInfo.m_officialComparisonResult.m_resultCode = ImageComparisonResult::ResultCode::FileNotFound;
         }
         else
         {
-            bool imagesWereCompared = DiffImages(
-                screenshotTestInfo.m_officialComparisonResult,
-                screenshotTestInfo.m_officialBaselineScreenshotFilePath,
+            bool imagesWereCompared = false;
+            AZ::Render::FrameCaptureRequestBus::BroadcastResult(
+                imagesWereCompared,
+                &AZ::Render::FrameCaptureRequestBus::Events::CompareScreenshots,
                 screenshotTestInfo.m_screenshotFilePath,
-                TraceLevel::Error);
+                screenshotTestInfo.m_officialBaselineScreenshotFilePath,
+                &screenshotTestInfo.m_officialComparisonResult.m_standardDiffScore,
+                &screenshotTestInfo.m_officialComparisonResult.m_filteredDiffScore,
+                ImperceptibleDiffFilter
+            );
+            screenshotTestInfo.m_officialComparisonResult.m_finalDiffScore = screenshotTestInfo.m_officialComparisonResult.m_standardDiffScore;
```

Comment:
I think we should preserve the comment that was in the original DiffImages function:
```
// Set the final score to the standard score just in case the filtered one is ignored
```

Reply:
Also, is ScriptReporter::DiffImages still used? Should it be removed now?

Reply:
Comment added back. Unused functions removed.